docs/geedocs/5.2.0/css/containers.css

```diff
@@ -0,0 +1,98 @@
+/*@charset "UTF-8";
```

Comment:
Just double checking - this is duplicated because it is also used by the main index.html?